packages/app/components/EventOverview/EventOverviewText.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { StyleSheet, Text, View } from "react-native";
+import GroupIcon from "../GroupScreen/GroupIcon";
+import PropTypes from "prop-types";
+
+const GRAY = "grey";
+const iconSize = 40;
+
+const styles = StyleSheet.create({
+  description: {
+    fontSize: 16,
+    marginTop: 10,
+  },
+  eventTitle: {
+    fontSize: 20,
+    fontWeight: "bold",
+    marginTop: 20,
+  },
+  groupIcon: {
+    flex: 1.2,
+  },
+  groupName: {
+    flex: 7,
+    fontSize: 19,
+    fontWeight: "bold",
+    marginRight: 5,
+    marginTop: 5,
+  },
+  overviewHeader: {
+    flexDirection: "row",
+  },
+  smallText: {
+    color: GRAY,
+    fontSize: 16,
+    marginTop: 10,
+  },
+});
+
+const OverviewHeader = ({ groupName }) => {
+  return (
+    <View style={styles.overviewHeader}>
+      <View style={styles.groupIcon}>
+        <GroupIcon
+          testID="groupIcon"
+          icon={require("@app/assets/test-club-icon.png")}
+          size={iconSize}
+          backgroundColor="#0E131F"
+        />
+      </View>
+      <Text style={styles.groupName}>{groupName}</Text>
+    </View>
+  );
+};
+
+export default function EventOverviewText(props) {
+  return (
+    <View>
+      <OverviewHeader groupName={props.groupName} />
+      <Text style={styles.eventTitle}>{props.title}</Text>
+      <Text style={styles.smallText}>
+        {props.timeBegin} - {props.timeEnd}
```

Comment:
The startTime appears as 2023-11-28T21:00:30.000Z - 2023-11-21T21:00:30.000Z so it can't read that well. If you can try to reformat so its a little more readable to something like 11/28/23 at 9:00 PM or November 28, 2023 at 9:00 PM instead of a direct timestamp.